src/v2i-hub/SpatPlugin/src/SpatMode.cpp

```diff
@@ -0,0 +1,21 @@
+#include "SpatMode.h"
+
+namespace SpatPlugin {
+    SPAT_MODE spat_mode_from_string(const std::string &mode) {
```

Comment:
Let us be consistent with the naming convention for a function, `camelCase`.

Reply:
I do not think camel case makes sense for enumerations since almost all the enumerations we have in V2X Hub use all capital letters like (tmx::utils::rsu::RSU_SPEC::RSU_4_1)